cineast-api/src/main/java/org/vitrivr/cineast/api/rest/resolvers/FileSystemThumbnailResolver.java

```diff
@@ -35,7 +35,7 @@ public ResolutionResult resolve(String segmentId) {
     File[] candidates = dir.listFiles(new FilenameFilter() {
       @Override
       public boolean accept(File dir, String name) {
-        return name.startsWith(segmentId);
+        return name.startsWith("shot" + split[1] + "_" + split[2]);
```

Comment:
While I haven't used these resolvers in the past, I'm not sure why these changes are required. Could you add a small comment, so we can check if the documentation or the default config requires an update?

Reply:
This might be able to be removed again. It was needed to be compatible with the format of the v3c dataset. Maybe something like this should be more easily configurable.